src/test/ensemble_config_change_stress.cpp

```diff
@@ -200,9 +200,10 @@ TEST_F(ConfigChangeStressTest, DISABLED_GetMetricsDuringLoad) {
 TEST_F(StressPipelineConfigChanges, RemoveDefaultVersionDuringPredictLoad) {
     std::set<StatusCode> requiredLoadResults = {StatusCode::OK,
         StatusCode::PIPELINE_DEFINITION_NOT_LOADED_YET,  // we hit when all config changes finish to propagate
-        StatusCode::MODEL_VERSION_NOT_LOADED_ANYMORE,    // we hit default version which is unloaded already but default is not changed yet
         StatusCode::MODEL_VERSION_MISSING};              // there is no default version since all are either not loaded properly or retired
-    std::set<StatusCode> allowedLoadResults = {};
+    std::set<StatusCode> allowedLoadResults = {
+        StatusCode::MODEL_VERSION_NOT_LOADED_ANYMORE,  // we hit default version which is unloaded already but default is not changed yet
+    };
```

Comment:
Why is it now allowed?

Reply:
It was always allowed - it is not required anymore - on slower machines we could sometimes not hit that state.